### PR TITLE
sumologicexporter: update labels -> attributes

### PR DESCRIPTION
**Pull request by codeboten**
**Description:**
Following the changes in core to OTLP 0.9.0 to move away from `Labels` and on to `Attributes`

**Link to tracking Issue:** Part of https://github.com/open-telemetry/opentelemetry-collector/issues/3535

### CODE REVIEW

**Comment by tigrannajaryan** [comment]
@pmm-sumo @sumo-drosiek please review.

File: exporter/sumologicexporter/prometheus_formatter.go [hunk 1]
```diff
@@ -55,11 +55,11 @@ func newPrometheusFormatter() (prometheusFormatter, error) {
...
 
 // PrometheusLabels returns all attributes as sanitized prometheus labels string
-func (f *prometheusFormatter) tags2String(attr pdata.AttributeMap, labels pdata.StringMap) prometheusTags {
+func (f *prometheusFormatter) tags2String(attr pdata.AttributeMap, labels pdata.AttributeMap) prometheusTags {
 	mergedAttributes := pdata.NewAttributeMap()
 	attr.CopyTo(mergedAttributes)
-	labels.Range(func(k string, v string) bool {
-		mergedAttributes.UpsertString(k, v)
+	labels.Range(func(k string, v pdata.AttributeValue) bool {
+		mergedAttributes.UpsertString(k, v.StringVal())
```
Comment (tigrannajaryan) [inline exporter/sumologicexporter/prometheus_formatter.go:62]: TODO link to https://github.com/open-telemetry/opentelemetry-collector/issues/3815

**Review by tigrannajaryan — APPROVED** [review]
LGTM

**Review by sumo-drosiek — APPROVED** [review]
LGTM

**Pull request merged**